baseplate/diagnostics/tracing/publisher.py

```diff
@@ -44,16 +42,15 @@ def __init__(self,
             zipkin_api_url,
             metrics_client,
             post_timeout=POST_TIMEOUT_DEFAULT,
-            zipkin_api_version=ZIPKIN_API_VERSION_DEFAULT,
-            zipkin_scheme=ZIPKIN_SCHEME_DEFAULT,
             retry_limit=RETRY_LIMIT_DEFAULT,
             num_conns=5):
 
         adapter = requests.adapters.HTTPAdapter(pool_connections=num_conns,
                                                 pool_maxsize=num_conns)
+        parsed_url = urlparse(zipkin_api_url)
         self.session = requests.Session()
-        self.session.mount("{}://".format(zipkin_scheme), adapter)
-        self.endpoint = "{}/spans".format(zipkin_api_url.rstrip('/'), zipkin_api_version)
+        self.session.mount("{}://".format(parsed_url.scheme), adapter)
+        self.endpoint = "{}{}/spans".format(parsed_url.netloc, parsed_url.path)
```

Comment:
should this have the scheme too?

Reply:
Yes, thank you. Added a simple test to check ``self.endpoint`` as well as the args to ``self.session.mount``.